src/about/index.html

```diff
@@ -13,7 +13,7 @@ <h1 class="intro-section__headline">About Us</h1>
   <div class="section-content">
     <h1 class="about-page__heading">About DecodeMTL</h1>
     <p class="about-page__text">
-      We are currently running our courses out of the GriffinCamp a leading startup accelerator in downtown Montreal's innovation district.
+      We are currently running our courses out of La Commune a leading startup accelerator in downtown Montreal's Old Port.
```

Comment:
La Commune is a co-working space rather than a startup accelerator. Can you please reflect that in the description?